website/source/docs/builders/virtualbox-iso.html.md

```diff
@@ -55,11 +55,11 @@ builder.
 ### Required:
 
 -   `iso_checksum` (string) - The checksum for the OS ISO file. Because ISO
-    files are so large, this is required and Packer will verify it prior to
-    booting a virtual machine with the ISO attached. The type of the checksum is
-    specified with `iso_checksum_type`, documented below. At least one of
-    `iso_checksum` and `iso_checksum_url` must be defined. This has precedence
-    over `iso_checksum_url` type.
+    files are so large, it is highly recommended that you provide a value for
```

Comment:
I'm also feeling a bit blind. Could you point out where in the code we remove the checksum requirement?  Or are you just updating docs to reflect reality